src/server.h

```diff
@@ -246,6 +246,7 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 #define CMD_ALLOW_BUSY ((1ULL << 26))
 #define CMD_MODULE_GETCHANNELS (1ULL << 27) /* Use the modules getchannels interface. */
 #define CMD_TOUCHES_ARBITRARY_KEYS (1ULL << 28)
+#define CMD_CROSS_SLOT (1ULL << 29)
```

Comment:
We also need to add this flag in the function: addReplyFlagsForCommand() which is for COMMAND INFO  (https://valkey.io/commands/command-info/)